lib/Varien/Io/File.php

```diff
@@ -251,7 +255,9 @@ public function streamClose()
         if ($this->_streamLocked) {
             $this->streamUnlock();
         }
-        @fclose($this->_streamHandler);
+        if ($this->_streamHandler) {
```

Comment:
I still get **fclose(): supplied resource is not a valid stream resource** :'(

Reply:
`if (is_resource ($this->_streamHandler)) {` ?

Reply:
Yes, that's true.